pinot-query-runtime/src/main/java/org/apache/pinot/query/mailbox/channel/MailboxContentObserver.java

```diff
@@ -88,6 +71,8 @@ public void onNext(MailboxContent mailboxContent) {
           LOGGER.warn("Mailbox: {} already cancelled from upstream", mailboxId);
           cancelStream();
           break;
+        case FIRST_ERROR:
+          return;
```

Comment:
I added this enum to keep the same behavior in this class, which just aborted the execution when exceptions are found in a message.